applicationinsights.ts

```diff
@@ -242,6 +244,20 @@ export class Configuration {
         return Configuration;
     }
 
+    /**
+     * Sets the state of pre aggregated metrics tracking (enabled by default)
+     * @param value if true pre aggregated metrics will be collected every minute and sent to Application Insights
+     * @returns {Configuration} this class
+     */
+    public static setAutoCollectPreAggregatedMetrics(value: boolean) {
```

Comment:
Curious, this api is exposed to users to enable/disable the feature?

Reply:
Yes that is why this API is exposed, similar approach is used for other auto collection features